btree/btree.c

```diff
@@ -1194,6 +1213,25 @@ static int node_valsize(const struct nd *node)
 	M0_PRE(node_invariant(node));
 	return (node->n_type->nt_valsize(node));
 }
+
+/**
+ * If predict is set as true,
+ * 	function will return true if there is possibility of underflow else
+ * 	it will return false.
```

Comment:
Change to:
If predict is 'true' the function returns a possibility of underflow if another record is deleted from this node without addition of any more records.